tcms/kiwi_auth/tests/test_admin.py

```diff
@@ -188,3 +191,36 @@ def test_should_be_able_to_edit_a_non_default_group(self):
             '<input type="text" name="name" value="%s" class="vTextField"'
             ' maxlength="150" required id="id_name">' % self.group.name,
         )
+
+    def test_should_be_allowed_to_create_new_group_with_added_user(self):
+        self.assertFalse(self.tester.groups.filter(name=self.group.name).exists())
+
+        group_name = "TestGroupName"
+        response = self.client.post(
+            reverse("admin:auth_group_add"),
+            {"name": group_name, "users": [self.tester.id]},
+            follow=True,
+        )
+
+        self.assertContains(response, group_name)
+        self.assertContains(
+            response,
+            '<a href="%s">%s</a>'
+            % (reverse("admin:auth_group_change", args=[4]), group_name),
```

Comment:
args should contain group.pk, not be hard-coded. You can objects.get instead of filter.exists and then use that to get the ID.